connect-examples/v2/node_bookings/README.md

```diff
@@ -38,34 +38,35 @@ In addition to using the Bookings API, the application demonstrates how to integ
 
 3. Set your credentials:
-    1. You need a *.env* file at the top directory to provide credentials. You can copy the content in the *.env.example* file provided in the project and use it as a template
-    2. In the file:
-        1. Set `ENVIRONMENT` to `sandbox` (for testing) or `production`
-        2. Replace the placeholder texts of `SQUARE_ACCESS_TOKEN` and `SQUARE_LOCATION_ID` with your access token and your seller location ID, respectively, for the chosen environment.
+    a. You need an *.env* file at the top directory to provide credentials. You can copy the content in the *.env.example* file provided in the project and use it as a template.
+    b. In the file:
+        * Set `ENVIRONMENT` to `Sandbox` (for testing) or `Production`.
+        * Replace the placeholder texts of `SQUARE_ACCESS_TOKEN` and `SQUARE_LOCATION_ID` with your access token and your seller location ID, respectively, for the chosen environment.
 
     You can find your Square credentials in the Square Developer Dashboard. For more information, see [Getting Started](https://developer.squareup.com/docs/get-started#step-2-create-an-application).
 
-    **IMPORTANT:** You can use your own credentials to test the sample application. If you plan to make a version of this sample application available for other users, you must use the Square [OAuth API](https://developer.squareup.com/docs/oauth-api/overview) to safely manage access to Square accounts.
+    **IMPORTANT:** You can use your own credentials to test the sample application. If you plan to make a version of this sample application available to other users, you must use the Square [OAuth API](https://developer.squareup.com/docs/oauth-api/overview) to safely manage access to Square accounts.
 
 4. Open a terminal and run the following command to install the sample application's dependencies:
 
    `npm install`
 
 ### Set up your seller account
 
-To run this app, your seller account must have appointments enabled. The appointments service is free in the `sandbox` environment and incurs service charges in the `production` environment. We recommend using the `sandbox` environment for development and testing purposes.
+To run this application, your seller account must have appointments enabled. The appointments service is free in the Sandbox environment and may incur service charges in the Production environment. You should use the Sandbox environment for development and testing purposes.
 
-To set up the apointments service in `sandbox`, follow these steps:
+To set up the appointments service in the Sandbox, follow these steps:
 
 #### 1. Initialize test data
 
-1. Ensure your `.env` file has `ENVIRONMENT` set to `sandbox` and provide sandbox environment values for `SQUARE_ACCESS_TOKEN` and `SQUARE_LOCATION_ID`
+a. Ensure that your *.env* file has `ENVIRONMENT` set to `Sandbox` and provides the Sandbox environment values for `SQUARE_ACCESS_TOKEN` and `SQUARE_LOCATION_ID`.
```

Comment:
I think the `Sandbox` value is [supposed to be lower-case](https://github.com/square/square-nodejs-sdk/blob/master/src/configuration.ts#L14-L16): `sandbox`

Reply:
Same with line 43 above

Reply:
- [ ] Fix `ENVIRONMENT` variable casing